systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -185,11 +186,23 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
                         .withNewInlineLogging()
                             .addToLoggers("rootLogger.level", "DEBUG")
                         .endInlineLogging()
+                        .withReadinessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
+                        .withLivenessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
                     .endUserOperator()
                     .editTopicOperator()
                         .withNewInlineLogging()
                             .addToLoggers("rootLogger.level", "DEBUG")
                         .endInlineLogging()
+                        .withReadinessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
+                        .withLivenessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
```

Comment:
Same as above ... should this really be here? Does the TO now with your changes really need 2 minutes to start? I do not see to be changed for the regular just for the STs which suggests it should not be here.

Reply:
No idea what's going on, but I saw in the long that KafkaStreamsTopicStoreService took ~1min to start (from "Starting" log to "Started").
It does a simple Kafka AdminClient check + starts Kafka Streams (topology),
which I really don't see why it should take so long ...

Reply:
If this is expected from the _new_ Topic Operator, then it certainly needs to be increased for all other places as well. Not just for running the system tests.

Reply:
It's not expected, it should be fast -- not as fast as ZKTopicStore, which didn't have anything to pre-start,
but few AdminClient calls + starting up Kafka Streams shouldn't take 1min.